attributes/master.rb

```diff
@@ -135,6 +135,11 @@
   master['use_system_accounts'] = true
 
   #
+  # The groups of user under which Jenkins is running. Works for runit only.
+  #
+  master['groups'] = [node['jenkins']['master']['group']]
```

Comment:
Yeah we should probably place things in the `runit` namespace then. much like we did for `sv_timeout`: https://github.com/opscode-cookbooks/jenkins/blob/159398823af39f085f33c169801901b7d0171f07/attributes/master.rb#L187-L193